library/src/math/align.rs

```diff
@@ -16,34 +16,29 @@ impl LayoutMath for AlignPointElem {
 
 /// Determine the position of the alignment points.
 pub(super) fn alignments(rows: &[MathRow]) -> Vec<Abs> {
-    let count = rows
-        .iter()
-        .map(|row| {
-            row.iter()
-                .filter(|fragment| matches!(fragment, MathFragment::Align))
-                .count()
-        })
-        .max()
-        .unwrap_or(0);
+    let mut points = Vec::<Abs>::new();
```

Comment:
I had a bit of trouble understanding that this vector now holds the widths first, maybe we can rename it to `widths` here and then rebind `let mut points = widths` before the second pass?

Reply:
Good point, done.